Week3/solution week-3/program 7.py

```diff
@@ -0,0 +1,11 @@
+import sqlite3
+conn=sqlite3.connect("c:/Sqlite/ESM.db")
+cur=conn.cursor()
+query="select * from Employee\

```

Comment:
What is the use of that last '\' (backward slash) or it is compulsory to write  '\' (backward slash) their

Reply:
It ignore space at end 9f line and it is not compulsory (if write query in triple single quote or triple double quotes)